spring-integration-file/src/main/java/org/springframework/integration/file/tail/OSDelegatingFileTailingMessageProducer.java

```diff
@@ -110,7 +109,7 @@ private void runExec() {
 			logger.info("Starting tail process");
 		}
 		try {
-			Process process = Runtime.getRuntime().exec(this.command);
+			Process process = Runtime.getRuntime().exec(getCommand());
```

Comment:
That's the way that I started to approach it, but I thought the test and the implementation would be cleaner/simpler if a start/stop wasn't required.  That said, I do appreciate consistency.  I'll fix this up to look more like `ApacheCommonsFileTailingMessageProducer` and will resubmit the patch.